packages/pangea-sdk/PangeaCyber.Net/Config.cs

```diff
@@ -40,37 +42,44 @@ public Config(string token, string domain)
 
         #region Public Properties
 
-        ///
+        /// <summary>Pangea API token.</summary>
         public string Token { get; set; }
 
-        ///
+        /// <summary>Pangea API domain.</summary>
         public string Domain { get; set; }
 
-        /// @deprecated set it on Service builder instead
+        /// <exclude />
+        [Obsolete("Set config ID when building service clients instead.")]
         public string ConfigID { get; set; } = default!;
 
-        ///
+        /// <summary>Pangea environment.</summary>
+        /// <remarks>
+        /// This is intended to facilitate SDK development and should not be touched in everyday usage.
+        /// </remarks>
         public string Environment { get; set; }
 
-        ///
-        public bool Insecure { get; set; } = default!;
+        /// <summary>Whether or not to perform requests via plain HTTP, as opposed to secure HTTPS.</summary>
+        public bool Insecure { get; set; }
 
-        ///
-        public TimeSpan ConnectionTimeout { get; set; } = default!;
+        /// <summary>Time span to wait before a request times out.</summary>
+        public TimeSpan ConnectionTimeout { get; set; }
 
-        ///
+        /// <summary>User-Agent string to append to the default one.</summary>
```

Comment:
do we append to the user-agent string in all our SDKs? do we not give the option to override?

Reply:
Just checked them all and yeah, append only.

Reply:
okay, as long as they're all the same I'm cool